help.md

```diff
@@ -22,15 +22,13 @@ Additionally the plugin can be installed by manually building the hpi file and u
 
 ## Technical Details
 
-### Development
```

Comment:
Why remove this section? It's there for plugin developers (both without rapid7 and the community)

Reply:
Under Contributions of the help.md (used by the extension listing library) we mention referencing the README/repo. Trying to keep developer specific content from the actual listing as it is intended for end users.

Reply:
Gotcha, the development section in help file has us covered